src/Control/Effect/Resumable.hs

```diff
@@ -82,7 +86,7 @@ runResumable :: ResumableC err m a -> m (Either (SomeError err) a)
 runResumable = runError . runResumableC
 
 newtype ResumableC err m a = ResumableC { runResumableC :: ErrorC (SomeError err) m a }
-  deriving newtype (Alternative, Applicative, Functor, Monad, MonadFail, MonadIO, MonadPlus, MonadTrans)
```

Comment:
I’d prefer the explicit deriving strategy. Leaving it tacit can introduce surprising bugs and helps me keep less in my head.

Reply:
Fair, tho I’d note we do have warnings for those turned on.